modules/appeals_api/spec/docs/v2/sc_spec.rb

```diff
@@ -200,8 +200,10 @@
       tags 'Supplemental Claims'
       operationId 'postSupplementalClaimEvidenceSubmission'
       description <<~DESC
-        This is the first step to submitting supporting evidence for an SC.  (See the Evidence Uploads section above for additional information.)
+        This is the first step to submitting supporting evidence for a Supplemental Claim.  (See the Evidence Uploads section above for additional information.)
         The Supplemental Claim GUID that is returned when the SC is submitted, is supplied to this endpoint to ensure the SC is in a valid state for sending supporting evidence documents.
```

Comment:
Not a blocker for this ticket, but there are still mixed uses of `SC` and `Supplemental Claims` in this description. Let's be sure to do a round of abbreviation cleanup at some point soon.